client/src/components/UI/SurveySnackbar.js

```diff
@@ -0,0 +1,176 @@
+import * as React from "react";
+import { useState } from "react";
+import { IconButton, Collapse, Button } from "@mui/material";
+import CloseIcon from "@mui/icons-material/Close";
+import Stack from "@mui/material/Stack";
+import Snackbar from "@mui/material/Snackbar";
+import Box from "@mui/material/Box";
+import ExpandLessIcon from "@mui/icons-material/ExpandLess";
+import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
+import { margin } from "@mui/system";
+
+const SurveySnackbar = () => {
+  const [open, setOpen] = useState(() => {
+    const isClosed = localStorage.getItem("surveySnackbarClosed");
+    return isClosed !== "true";
+  });
+  const [expanded, setExpanded] = useState(false);
+
+  const handleClose = (event, reason) => {
+    if (reason === "clickaway") {
+      return;
+    }
+    setOpen(false);
+    localStorage.setItem("surveySnackbarClosed", "true");
+  };
+
+  const toggleExpansion = () => {
+    setExpanded((prevExpanded) => !prevExpanded);
+  };
+
+  const openForm = () => {
+    window.open(
+      "https://docs.google.com/forms/d/e/1FAIpQLSdAhi_nMKQfWVHjfpl1ZkmymBTt8BW7YNqVIOJ4JKYgSL4O3g/viewform",
+      "_blank"
+    );
+  };
+
+  const labelStyles = {
+    marginBottom: "10px",
```

Comment:
as much as possible, we want to avoid adding these margins on specific elements. If we need to add some spacing within elements, we can use a `<Stack />` instead. It's cleaner and easier to maintain 👌

Reply:
oh nvm, this isn't being used anymore lol we can remove it 😂